.github/workflows/build.yml

```diff
@@ -0,0 +1,245 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  native-build-linux-x86-64:
+    runs-on: ubuntu-latest
+    env:
+      SKIP_GRADLE: true
+    steps:
+      - name: Prepare
+        run: |
+          # secp256k1 dependencies
+          sudo apt-get update
+          sudo apt-get install -y autoconf build-essential libtool automake patchelf
+          # rust dependencies
+          export CARGO_HOME="$HOME/.cargo"
+          curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain 1.62.1
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Build
+        run: ./build.sh
+      - uses: actions/upload-artifact@v3.1.0
+        with:
+          name: altbn128 native build artifacts
+          path: altbn128/build/
+      - uses: actions/upload-artifact@v3.1.0
+        with:
+          name: blake2bf native build artifacts
+          path: blake2bf/build/
+      - uses: actions/upload-artifact@v3.1.0
+        with:
+          name: bls12-381 native build artifacts
+          path: bls12-381/build/
+      - uses: actions/upload-artifact@v3.1.0
+        with:
+          name: secp256k1 native build artifacts
+          path: secp256k1/build/
+      - uses: actions/upload-artifact@v3.1.0
+        with:
+          name: secp256r1 native build artifacts
+          path: secp256r1/besu-native-ec/release/
+      - uses: actions/upload-artifact@v3.1.0
+        with:
+          name: ipa-multipoint native build artifacts
+          path: ipa-multipoint/build/
+  native-build-linux-arm64:
+    runs-on: besu
```

Comment:
Can we get a different name?  This to me implies a server with besu on it, no a build environment. is it ok if it's named `besu-ci` like the github actions config?

Reply:
You might be able to change in github in the runner tags. I cannot access the admin interface.